### PR TITLE
chan!(*): `name` -> `pkgname`

### PR DESCRIPTION
**Pull request by Elsie19**
## Purpose

Conform with `PKGBUILD`.

## Checklist

- [x] I confirm that I have read the [contributing guidelines](https://github.com/pacstall/pacstall/blob/develop/CONTRIBUTING.md), and this pull request is abiding by all the clauses stated in the guideline.

### CODE REVIEW

File: misc/scripts/install-local.sh [hunk 1]
```diff
@@ -30,36 +30,36 @@ source "${STGDIR}/scripts/checks.sh" || {
...
     fi
     if [[ -f "/tmp/pacstall-pacdeps-$PACKAGE" ]]; then
         sudo rm -rf "/tmp/pacstall-pacdeps-$PACKAGE"
         sudo rm -rf /tmp/pacstall-pacdep
     else
         sudo rm -rf "${SRCDIR:?}"/*
-        # just in case we quit before $name is declared, we should be able to remove a fake directory so it doesn't exit out the script
-        sudo rm -rf "${STOWDIR:-/usr/src/pacstall}/${name:-raaaaaaaandom}"
+        # just in case we quit before $pkgname is declared, we should be able to remove a fake directory so it doesn't exit out the script
+        sudo rm -rf "${STOWDIR:-/usr/src/pacstall}/${pkgname:-raaaaaaaandom}"
```
Comment (oklopfer) [inline misc/scripts/install-local.sh:47]: Can this actually be random and not fake text that says random

**Review by oklopfer — CHANGES_REQUESTED** [review]

File: misc/scripts/install-local.sh (hunk 1 above)
Comment (oklopfer) [reply misc/scripts/install-local.sh:47]: or maybe just like `pacstall-MMDDYYYY-Thh-mm-ss`

Comment (Elsie19) [reply misc/scripts/install-local.sh:47]: wait I have an idea.

Comment (Elsie19) [inline misc/scripts/install-local.sh:47]:
```suggestion
        if [[ -n $pkgname ]]; then
            sudo rm -rf "${STOWDIR:-/usr/src/pacstall}/${pkgname}"
        fi
```

File: misc/scripts/install-local.sh [hunk 2]
```diff
@@ -30,36 +30,37 @@ source "${STGDIR}/scripts/checks.sh" || {
...
+        if [[ -n $pkgname ]]; then
+            sudo rm -rf "${STOWDIR:-/usr/src/pacstall}/${pkgname}"
+        fi
         rm -rf /tmp/pacstall-gives
     fi
-    sudo rm -rf "${STOWDIR}/${name:-$PACKAGE}.deb"
+    sudo rm -rf "${STOWDIR}/${pkgname:-$PACKAGE}.deb"
     rm -f /tmp/pacstall-select-options
-    unset name repology pkgver epoch url depends makedepends breaks replace gives pkgdesc hash optdepends ppa arch maintainer pacdeps patch PACPATCH NOBUILDDEP provides incompatible optinstall epoch homepage backup pkgrel mask pac_functions repo priority 2> /dev/null
+    unset pkgname repology pkgver epoch url depends makedepends breaks replace gives pkgdesc hash optdepends ppa arch maintainer pacdeps patch PACPATCH NOBUILDDEP provides incompatible optinstall epoch homepage backup pkgrel mask pac_functions repo 2> /dev/null
```
Comment (Elsie19) [inline misc/scripts/install-local.sh:53]:
```suggestion
    unset pkgname repology pkgver epoch url depends makedepends breaks replace gives pkgdesc hash optdepends ppa arch maintainer pacdeps patch PACPATCH NOBUILDDEP provides incompatible optinstall epoch homepage backup pkgrel mask pac_functions repo  priority 2> /dev/null
```

File: misc/scripts/install-local.sh [hunk 3]
```diff
@@ -30,36 +30,37 @@ source "${STGDIR}/scripts/checks.sh" || {
...
+        if [[ -n $pkgname ]]; then
+            sudo rm -rf "${STOWDIR:-/usr/src/pacstall}/${pkgname}"
+        fi
         rm -rf /tmp/pacstall-gives
     fi
-    sudo rm -rf "${STOWDIR}/${name:-$PACKAGE}.deb"
+    sudo rm -rf "${STOWDIR}/${pkgname:-$PACKAGE}.deb"
     rm -f /tmp/pacstall-select-options
-    unset name repology pkgver epoch url depends makedepends breaks replace gives pkgdesc hash optdepends ppa arch maintainer pacdeps patch PACPATCH NOBUILDDEP provides incompatible optinstall epoch homepage backup pkgrel mask pac_functions repo priority 2> /dev/null
+    unset pkgname repology pkgver epoch url depends makedepends breaks replace gives pkgdesc hash optdepends ppa arch maintainer pacdeps patch PACPATCH NOBUILDDEP provides incompatible optinstall epoch homepage backup pkgrel mask pac_functions repo  priority 2> /dev/null
```
Comment (Elsie19) [inline misc/scripts/install-local.sh:53]:
```suggestion
    unset pkgname repology pkgver epoch url depends makedepends breaks replace gives pkgdesc hash optdepends ppa arch maintainer pacdeps patch PACPATCH NOBUILDDEP provides incompatible optinstall epoch homepage backup pkgrel mask pac_functions repo priority 2> /dev/null
```

**Review by Elsie19 — COMMENTED** [review]
meh

**Comment by oklopfer** [comment]
Screenshot 2024-02-23 at 9 06 46 PM

**Comment by Elsie19** [comment]
Yep.

**Comment by oklopfer** [comment]
```bash
/usr/share/pacstall/scripts/build-local.sh: line 60: name: parameter null or not set
pacstall@docker-desktop:~$ 
```
hm

**Comment by oklopfer** [comment]
```bash
[+] INFO: Packaging renpy-full as renpy
	[>] Packing control.tar
	[>] Packing data.tar
	[>] Compressing
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'renpy' instead of '/usr/src/pacstall/renpy-full.deb'
The following NEW packages will be installed:
  renpy
0 upgraded, 1 newly installed, 0 to remove and 9 not upgraded.
Need to get 0 B/810 B of archives.
After this operation, 4096 B of additional disk space will be used.
Get:1 /usr/src/pacstall/renpy-full.deb renpy arm64 8.1.3-pacstall1 [810 B]
Selecting previously unselected package renpy.
(Reading database ... 15868 files and directories currently installed.)
Preparing to unpack .../src/pacstall/renpy-full.deb ...
Unpacking renpy (8.1.3-pacstall1) ...
Setting up renpy (8.1.3-pacstall1) ...
[+] INFO: Performing post install operations
	[>] Storing pacscript
	[>] Cleaning up
[+] INFO: Done installing renpy-full
pacstall@docker-desktop:~$ pacstall -Qi renpy-full
name: renpy
version: 8.1.3-pacstall1
size: 4
description: A visual novel engine
date installed: Sat Mar  2 08:34:57 PM Africa 2024
homepage: https://www.renpy.org
maintainer: vigress8 <vig@disroot.org>
install type: explicitly installed
pacstall@docker-desktop:~$ 
```
cool works now

**Review by oklopfer — DISMISSED** [review]

**Review by oklopfer — APPROVED** [review]

**Pull request merged**